src/control/include/pure_pursuit/pp.hpp

```diff
@@ -0,0 +1,112 @@
+#include "common_lib/structures/structures.hpp"
+#include "custom_interfaces/msg/path_point_array.hpp"
+#include "custom_interfaces/msg/pose.hpp"
+#include "rclcpp/rclcpp.hpp"
+
+/**< Wheel base of the vehicle in m */
+constexpr double WHEEL_BASE = 1.5;
+
+/**< Distance from the center of gravity to the rear axis in m */
+constexpr double DIST_CG_2_REAR_AXIS = 0.5;
```

Comment:
For now this is ok, but I also refactored this to the common_lib. Same thing with the Point and Pose as I also extracted it. But we can merge it like this and then apply fixes, or either your Pr or mine wait for the other and then rebase